### PR TITLE
Price compare block

### LINKED ISSUE #195: Product card variation for comparison with other products
Product card variation for the comparison with other products 
Found on page https://www.bitdefender.com.au/solutions/vpn.html

image

### PR DESCRIPTION
**Pull request by andreibogdan**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #195 

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/vpn
- After: https://195-product-card-variations--bitdefender--hlxsites.hlx.page/solutions/vpn

### CODE REVIEW

File: blocks/products/products.css [hunk 1]
```diff
@@ -390,4 +390,161 @@
...
+    padding: 30px 30px 30px 48px;
+    z-index: 2;
+  }
+
+  .products.compare .product-card h4 {
+    font-size: var(--body-font-size-l);
+  }
+
+  .products.compare .product-card ul li {
+    font-size: 20px;
```
Comment (sergiucoman) [inline blocks/products/products.css:547]: Font on mobile is a bit bigger than in the original site.

**Review by sergiucoman — COMMENTED** [review]
The percentage of the save is not displayed (not sure if related to this or not).

image

**Comment by andreibogdan** [comment]
> The percentage of the save is not displayed (not sure if related to this or not).
> 
> image

The discount issue will be addressed separately. @nicolasbarbe is working on this.

File: blocks/products/products.css (hunk 1 above)
Comment (andreibogdan) [reply blocks/products/products.css:618]: Fixed.

**Review by sergiucoman — APPROVED** [review]

**Pull request merged**